dotcom-rendering/src/web/components/LightboxPicture.tsx

```diff
@@ -0,0 +1,122 @@
+		{ breakpoint: breakpoints.desktop, width: breakpoints.desktop },
+		{ breakpoint: breakpoints.tablet, width: breakpoints.tablet },
+		{ breakpoint: breakpoints.phablet, width: breakpoints.phablet },
+		{
+			breakpoint: breakpoints.mobileLandscape,
+			width: breakpoints.mobileLandscape,
+		},
+		{
+			breakpoint: breakpoints.mobileMedium,
+			width: breakpoints.mobileMedium,
+		},
+		{ breakpoint: breakpoints.mobile, width: breakpoints.mobile },
+	];
+};
+
+/**
+ * Used on `picture` and `img` to prevent having a line-height,
+ * as these elements are which are `inline` by default.
+ *
+ * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#styling_with_css
+ */
+const block = css`
+	display: block;
+`;
+
+/** On fronts, all images are 5:3 */
+// const aspectRatio = css`
+// 	padding-top: ${(3 / 5) * 100}%;
+// 	position: relative;
+// 	& > * {
+// 		position: absolute;
+// 		top: 0;
+// 		left: 0;
+// 		width: 100%;
+// 		height: 100%;
+// 	}
+// `;
+
+export const LightboxPicture = ({ master, alt }: Props) => {
+	const imageWidths = decideImageWidths();
+	const sources = imageWidths
+		.slice()
+		.sort(descendingByBreakpoint)
+		.map(({ width: imageWidth, breakpoint }) => {
+			return {
+				breakpoint,
+				width: imageWidth,
+				hiResUrl: generateImageURL({
+					master,
+					imageWidth,
+					resolution: 'high',
+				}),
+				lowResUrl: generateImageURL({
+					master,
+					imageWidth,
+					resolution: 'low',
+				}),
+			};
+		});
+
+	/**
+	 * The assumption here is readers on devices that do not support srcset
+	 * are likely to be on poor network connections so we're going
+	 * to fallback to the smallest image.
+	 *
+	 * Sources are ordered in `descendingByBreakpoint` order,
+	 * so the last one is the smallest.
+	 */
+	const [fallbackSource] = sources.slice(-1);
+
+	return (
+		<picture css={[block]}>
+			{sources.map((source) => {
+				return (
+					<React.Fragment key={source.breakpoint}>
+						{/* High resolution (HDPI) sources*/}
+						<source
+							srcSet={source.hiResUrl}
+							media={`(min-width: ${source.breakpoint}px) and (-webkit-min-device-pixel-ratio: 1.25), (min-width: ${source.breakpoint}px) and (min-resolution: 120dpi)`}
+						/>
+						{/* Low resolution (MDPI) source*/}
+						<source
+							srcSet={source.lowResUrl}
+							media={`(min-width: ${source.breakpoint}px)`}
+						/>
+					</React.Fragment>
+				);
+			})}
+
+			<img
+				alt={alt}
+				// The assumption here is readers on devices that do not support srcset are likely to be on poor
+				// network connections so we're going to fallback to the smallest image
+				src={fallbackSource.lowResUrl}
+				// height={fallbackSource.hei / ratio}
+				css={block}
+			/>
+		</picture>
+	);
+};
+
+// We use disableLazyLoading to decide if we want to turn off lazy loading of images site wide. We use this
+// to prevent false negatives on Chromatic snapshots (see /.storybook/config)
+LightboxPicture.disableLazyLoading = false;
```

Comment:
If we're not using the `lazy` prop on the `img` then we don't need this

Reply:
I think I want to look into whether the the lazy prop would be useful for us - the mobile experience isn't an area I've looked at yet but it might change things up a bit!

Reply:
`loading=lazy` on an image defers the download until the browser [thinks the image is needed](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading) (based on if it is in the viewport or not)

> Tells the user agent to hold off on loading the image until the browser estimates that it will be needed imminently. For instance, if the user is scrolling through the document, a value of lazy will cause the image to only be loaded shortly before it will appear in the window's [visual viewport](https://developer.mozilla.org/en-US/docs/Glossary/Visual_Viewport).

This isn't 100% clear but I think this means that if an `img` is off page, as, a `dialog` is not visible, then the image will not be loaded. 

There's also the question of these images affecting the page load event

> The [load](https://developer.mozilla.org/en-US/docs/Web/API/Window/load_event) event is fired when the document has been fully processed. When images are loaded eagerly (which is the default), every image in the document must be fetched before the load event can fire.
> 
> By specifying the value lazy for loading, you prevent the image from delaying the load attribute by the amount of time it takes to request, fetch, and process the image.

None of this conclusively makes the case for using `loading-lazy` but my feeling is we probably want to at least test and investigate using it